File: ruby/jelly/const.rb
module Jelly
    EMPTY_CHAR = '.'
    WALL_CHAR = '#'
    RE_JELLY_CHARS = /[RBYG]/
    BLACK = '@'

    DIRS = {
        '<' => [-1, 0],
        '>' => [1, 0],
        '^' => [0, -1],
        'v' => [0, 1],
    }
end

File: java/jelly/Const.java
package jelly;

import java.util.Map;

public class Const {
    public static final char EMPTY_CHAR = '.';
    public static final char WALL_CHAR = '#';
    public static final String RE_JELLY_CHARS = "[RBYG]";
    public static final char BLACK = '@';

    public static final Map<Character, int[]> DIRS = Map.of(
        '<', new int[]{-1, 0},
        '>', new int[]{1, 0},
        '^', new int[]{0, -1},
        'v', new int[]{0, 1}
    );
}
